build.sh

```
@@ -0,0 +1,2 @@
+#!/bin/bash
+env $(cat .env.local | xargs) ./node_modules/.bin/next build
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)